app/jobs/google/search_keyword_job.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Google
+  class SearchKeywordJob < ApplicationJob
+    queue_as :default
```

Comment:
```suggestion
```

Because it's default, i think we can remove this line. Explicit is good i think, however if it's a convention in Rails, then we can eliminate this to avoid typing mistake.